### PR TITLE
Implement label wrapping on wxInfoBar GTK and show it in the sample

### PR DESCRIPTION
**Pull request by oneeyeman1**
(Sorry about wasting the PR. I picked the wrong branch and didn't realize).

This implements text wrapping on the wxInfoBar for GTK+ native version. Sample is also updated to easily test the functionality.

I tried to check the generic version, but I guess there is a code that works around this ticket by wrapping the code by explicit \n' character.

I hope someone will be able to fix the generic version and make the label of wxInfoBar wrap.

### CODE REVIEW

**Comment by oneeyeman1** [comment]
Just adding generic implementation to test proper wrapping.

File: src/gtk/infobar.cpp [hunk 1]
```diff
@@ -196,7 +196,9 @@ void wxInfoBar::ShowMessage(const wxString& msg, int flags)
     if ( wxGTKImpl::ConvertMessageTypeFromWX(flags, &type) )
         gtk_info_bar_set_message_type(GTK_INFO_BAR(m_widget), type);
     gtk_label_set_text(GTK_LABEL(m_impl->m_label), wxGTK_CONV(msg));
-
+    gtk_label_set_line_wrap(GTK_LABEL(m_impl->m_label), TRUE );
+    if( gtk_check_version( 2, 10, 0 ) )
```
Comment (vadz) [inline src/gtk/infobar.cpp]: You need to add compile-time check too and should use `wx_is_at_least_gtk2()`. Look at any other place where GTK version is checked.

File: src/generic/infobar.cpp [hunk 2]
```diff
@@ -236,7 +236,7 @@ void wxInfoBarGeneric::ShowMessage(const wxString& msg, int flags)
     // notice the use of EscapeMnemonics() to ensure that "&" come through
     // correctly
     m_text->SetLabel(wxControl::EscapeMnemonics(msg));
-
+	m_text->Wrap( GetClientSize().GetWidth() );
```
Comment (vadz) [inline src/generic/infobar.cpp]:
```suggestion
    m_text->Wrap( GetClientSize().GetWidth() );
```
(removing hard TAB...)

File: src/gtk/infobar.cpp [hunk 3]
```diff
@@ -196,7 +196,9 @@ void wxInfoBar::ShowMessage(const wxString& msg, int flags)
     if ( wxGTKImpl::ConvertMessageTypeFromWX(flags, &type) )
         gtk_info_bar_set_message_type(GTK_INFO_BAR(m_widget), type);
     gtk_label_set_text(GTK_LABEL(m_impl->m_label), wxGTK_CONV(msg));
-
+    gtk_label_set_line_wrap(GTK_LABEL(m_impl->m_label), TRUE );
```
Comment (vadz) [inline src/gtk/infobar.cpp]:
```suggestion
    gtk_label_set_line_wrap(GTK_LABEL(m_impl->m_label), TRUE);
```
You can use spaces or not, but not use them asymmetrically, this is really too weird.

**Review by vadz — CHANGES_REQUESTED** [review]
Please fix the GTK version check. TIA!

**Comment by oneeyeman1** [comment]
> 
> 
> Please fix the GTK version check. TIA!

All fixed.

Please check and apply.

TIA!

**Pull request closed without merging**